package.json

```diff
@@ -4,23 +4,121 @@
   "description": "Lambda Function to Stream Logs from AWS S3 to LogDNA",
   "main": "index.js",
   "scripts": {
-    "lint": "./node_modules/.bin/eslint -c .eslintrc *.js"
+    "docs": "config-doc lib/config.js > doc/env.md",
+    "lint": "eslint .",
+    "lint:fix": "npm run lint -- --fix",
+    "tap": "tap",
+    "pretest": "npm run lint",
+    "pretest:ci": "npm run lint",
+    "test": "npm run tap",
+    "test:ci": "tools/test-ci.sh",
+    "package": "tools/package.sh",
+    "release": "semantic-release"
+  },
+  "release": {
+    "branches": [
+      "master"
+    ],
+    "plugins": [
+      [
+        "@semantic-release/github",
+        {
+          "assets": [
+            {
+              "path": "pkg/logdna-s3.zip"
+            }
+          ]
+        }
+      ]
+    ],
+    "extends": "semantic-release-config-logdna"
+  },
+  "eslintConfig": {
+    "root": true,
+    "ignorePatterns": [
+      "node_modules/",
+      "coverage/"
+    ],
+    "extends": [
+      "logdna"
+    ],
+    "parserOptions": {
+      "ecmaVersion": 2019
+    }
+  },
+  "tap": {
```

Comment:
Remove `.taprc` if this block is in here.

Reply:
Removed!